examples/temperature/temperature-service-console-client/main.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016 Pelagicore AB
+ *
+ * Permission to use, copy, modify, and/or distribute this software for
+ * any purpose with or without fee is hereby granted, provided that the
+ * above copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+ * WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+ * BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+ * OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+ * WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+ * ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+ * SOFTWARE.
+ *
+ * For further information see LICENSE
+ */
+
+#include <thread>
+#include <unistd.h>
+#include <dbus-c++/dbus.h>
+
+#include "temperatureserviceinterface.h"
+#include "periodictemperature.h"
+
+int main()
+{
+    // Setup DBus
+    DBus::BusDispatcher dispatcher;
+    DBus::default_dispatcher = &dispatcher;
+    DBus::Connection bus = DBus::Connection::SystemBus();
+
+    // The interface class needs a path to a logfile because we
+    // can't access stdout.
+    std::string homeDir = getenv("HOME");
+    if (homeDir.empty()) {
+        std::cout << "No $HOME set, can't run" << std::endl;
```

Comment:
We don't need to write anywhere really, but to minimize dependencies (I guess) we don't pull ivi-logging in here.